packages/protocol/contracts/stability/GrandaMento.sol

```diff
@@ -33,17 +33,32 @@ contract GrandaMento is
     bool sellCelo
   );
 
+  // Emitted when an exchange proposal is approved by the approver.
+  event ExchangeProposalApproved(uint256 indexed proposalId);
+
+  // Emitted when an exchange proposal is cancelled.
+  event ExchangeProposalCancelled(uint256 indexed proposalId, address sender);
+
+  // Emitted when an exchange proposal is executed.
+  event ExchangeProposalExecuted(uint256 indexed proposalId);
+
+  // Emitted when the approver is set.
+  event ApproverSet(address approver);
+
   // Emitted when the spread is set.
   event SpreadSet(uint256 spread);
 
+  // Emitted when the veto period in seconds is set.
+  event VetoPeriodSecondsSet(uint256 vetoPeriodSeconds);
+
   // Emitted when the exchange limits for a stable token are set.
   event StableTokenExchangeLimitsSet(
-    address indexed stableToken,
+    string stableTokenRegistryId,
```

Comment:
Not indexed because an indexed string will actually emit the value equivalent to `keccak(abi.encodePacked(strValue))` rather than `strValue`. I figured that indexed events aren't *that* useful because presumably a chain indexing service will still be able to work regardless of the parameter being indexed or not.

An alternative is to emit both:
`string stableTokenRegistryId, bytes32 indexed stableTokenRegistryIdHash`